_pages/policies/detailing-and-offboarding-policies/leaving-tts.md

```diff
@@ -108,3 +108,53 @@ As you leave, you’re encouraged to join the [#alumni](https://gsa-tts.slack.co
 ### Writing a post on your personal blog
 
 If you want to write a post on your personal blog about your experiences in government or why you’re leaving, we recommend that you wait until you are no longer on staff before you hit “Publish.” This helps to avoid any issues with Hatch Act compliance and with inadvertently writing or speaking on behalf of GSA without permission.
+
+### Post-employment restrictions
```

Comment:
Question: this feels pretty legalese. Do we have leeway to rewrite it to be simpler and clearer, or do we need to keep the legal langauge. And if the latter, would we be better just having a graf and a link to an authoritative resource? Maybe something along the lines of "We understand as a term employee you'll eventually be seeking your next job. You need to understand as a government employee there are rules and regulations around what you can do as part of that job search. Read them." 

Thoughts?

Reply:
+100 — this feels very hard to read, and not in line with the tone of the Handbook otherwise. We should either paraphrase or link to the actual legal guidance (or both!).

Reply:
We are not free to rewrite this.

Reply:
Should we delete the stuff?

Reply:
Is there a canonical place it exists (InSite?) that we can link to instead?

Reply:
Agreed. I think we can plain language "Hey, this is important, and you should read it so you stay on the good side of the law" and then link to the legalese. I wouldn't want to paraphrase the legalese (I'd be too worried about missing some nuance) but I think it makes perfect sense to just say it's important and link to it.

Reply:
Alas this does not exist anywhere, according to OGC.  They wrote it for us.

Reply:
Done!